src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base.css.twig

```diff
@@ -0,0 +1,52 @@
+{# Normalization
+   (normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css)
+   ========================================================================= #}
+*{-webkit-box-sizing: border-box;-moz-box-sizing: border-box;box-sizing: border-box}html{font-family:sans-serif;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%}body{margin:0}article,aside,details,figcaption,figure,footer,header,hgroup,main,menu,nav,section,summary{display:block}audio,canvas,progress,video{display:inline-block;vertical-align:baseline}audio:not([controls]){display:none;height:0}[hidden],template{display:none}a{background-color:transparent}a:active,a:hover{outline:0}abbr[title]{border-bottom:1px dotted}b,strong{font-weight:700}dfn{font-style:italic}h1{margin:.67em 0;font-size:2em}mark{color:#000;background:#ff0}small{font-size:80%}sub,sup{position:relative;font-size:75%;line-height:0;vertical-align:baseline}sup{top:-.5em}sub{bottom:-.25em}img{border:0}svg:not(:root){overflow:hidden}figure{margin:1em 40px}hr{height:0;-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}pre{overflow:auto}code,kbd,pre,samp{font-family:monospace,monospace;font-size:1em}button,input,optgroup,select,textarea{margin:0;font:inherit;color:inherit}button{overflow:visible}button,select{text-transform:none}button,html input[type="button"],input[type="reset"],input[type="submit"]{-webkit-appearance:button;cursor:pointer}button[disabled],html input[disabled]{cursor:default}button::-moz-focus-inner,input::-moz-focus-inner{padding:0;border:0}input{line-height:normal}input[type="checkbox"],input[type="radio"]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;padding:0}input[type="number"]::-webkit-inner-spin-button,input[type="number"]::-webkit-outer-spin-button{height:auto}input[type="search"]{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;-webkit-appearance:textfield}input[type="search"]::-webkit-search-cancel-button,input[type="search"]::-webkit-search-decoration{-webkit-appearance:none}fieldset{padding:.35em .625em .75em;margin:0 2px;border:1px solid silver}legend{padding:0;border:0}textarea{overflow:auto}optgroup{font-weight:700}table{border-spacing:0;border-collapse:collapse}td,th{padding:0}
+
+:root {
```

Comment:
Including `normalize.css` and non-namespaced custom properties on `:root` is a total no-go if you import this file in the toolbar CSS (which you currently do). The toolbar CSS must be properly scoped to avoid conflicts with styles of the page, as it gets injected in the page. Otherwise, we would be changing the design of all projects using Symfony when injecting the debug toolbar (meaning that the only reliable way to work on CSS for a Symfony project would be to disable the web debug toolbar, which is bad DX).

Reply:
my proposal:

- keep the normalize.css styles in the profile.css.twig file (keeping the `.sf-toolbarreset *` styles that you removed to have border-box sizing on the toolbar)
- update all custom property names to have a `sf-` prefix
- create a template that contains the custom properties without the selector around them (or uses a variable to set the selector). Then, `profiler.css.twig` would include it in `:root` while `toolbar.css.twig` would include it in `.sf-toolbarreset` or `.sf-toolbar` (depending on which one is right for that).